src/debuginfo.rs

```diff
@@ -91,7 +91,7 @@ fn compute_mir_scopes<'gcc, 'tcx>(
 /// FIXME(tempdragon/?): Add Scope Support Here.
 fn make_mir_scope<'gcc, 'tcx>(
     cx: &CodegenCx<'gcc, 'tcx>,
-    instance: Instance<'tcx>,
+    _instance: Instance<'tcx>,
```

Comment:
Well, there are two reasons for this. 
1. You advised to keep it as in cg_llvm so that you can merge them somewhere in cg_ssa.
2. I haven't investigated into GCC &  libgccjit for ways to deal with scopes, and so I keep it here for potential use.